src/backend/Pg.cpp

```diff
@@ -1078,6 +1078,35 @@ CREATE TABLE IF NOT EXISTS account_transactions (
     PRIMARY KEY (account, ledger_seq, transaction_index, hash)
 ) PARTITION BY RANGE (ledger_seq);
 
+-- Table that stores NFTs, since they are found in pages and we need O(1) access to arbitrary ones.
+-- Deletes from the ledger table cascade here based on ledger_seq.
+CREATE TABLE IF NOT EXISTS nf_tokens (
+    token_id      bytea     NOT NULL,
+    ledger_seq    bigint    NOT NULL REFERENCES ledgers ON DELETE CASCADE,
+    issuer        bytea     NOT NULL,
```

Comment:
Do we also need a key for `issuer`? 

In cassandra we have a separate table mapping issuer to token_id?